elements/lisk-chain/test/unit/state_store_account.spec.ts

```diff
@@ -196,79 +194,14 @@ describe('state store / account', () => {
 		});
 	});
 
-	describe('set', () => {
```

Comment:
this should be replaced by some other property because it's checking the `set` functionality